### DISCUSSION TITLE
Feature Request: New event that fires after `artisan migrate:fresh`

### DISCUSSION THREAD

**Discussion by WyattCast44** [laravel/framework#48372] [Ideas]
# Feature Request: New event that fires after `artisan migrate:fresh`

Would anyone else get value from adding a new event that fires after the artisan `migrate:fresh` command finishes running. For my particular use case, I want to listen to the event and clean up any uploaded files from manual testing. 

I am happy to put together a PR if anyone else thinks this would be a meaningful DX improvement. Cheers!

**Comment by crynobone** [comment]
`migrate:fresh` should already fired https://github.com/laravel/framework/blob/10.x/src/Illuminate/Database/Events/DatabaseRefreshed.php event and should be suitable for your use case.
